autom8s/helm-server.js

```diff
@@ -1,66 +1,62 @@
-﻿'use strict';
-const HelmWrapper = require("./helm-wrapper");
-var express = require('express');
-var router = express.Router();
+﻿const express = require('express');
+const HelmWrapper = require('./helm-wrapper');
```

Comment:
`const HelmWrapper`
I think it's not a good idea to have a static object. why not instantiate it for each request?

Reply:
Fixed